code/modules/research/xenobiology/crossbreeding/burning.dm

```diff
@@ -0,0 +1,426 @@
+	var/amount = rand(3,6)
+	var/list/turfs = list()
+	for(var/turf/open/T in range(1,get_turf(user)))
+		turfs += T
+	for(var/i = 0, i < amount, i++)
+		var/path = get_random_food()
+		var/obj/item/O = new path(pick(turfs))
+		O.reagents.add_reagent("slimejelly",5) //Oh god it burns
+		if(prob(50))
+			O.desc += " It smells strange..."
+	user.visible_message("<span class='danger'>[src] produces a few pieces of food!</span>")
+	..()
+
+/obj/item/slimecross/burning/bluespace
+	colour = "bluespace"
+
+/obj/item/slimecross/burning/bluespace/do_effect(mob/user)
+	user.visible_message("<span class='danger'>[src] sparks, and lets off a shockwave of bluespace energy!</span>")
+	for(var/mob/living/L in range(1, get_turf(user)))
+		if(L != user)
+			do_teleport(L, get_turf(L), 6, asoundin = 'sound/effects/phasein.ogg') //Somewhere between the effectiveness of fake and real BS crystal
+			new /obj/effect/particle_effect/sparks(get_turf(L))
+			playsound(get_turf(L), "sparks", 50, 1)
+	..()
+
+/obj/item/slimecross/burning/sepia
+	colour = "sepia"
+
+/obj/item/slimecross/burning/sepia/do_effect(mob/user)
+	user.visible_message("<span class='notice'>[src] shapes itself into a camera!</span>")
+	new /obj/item/device/camera/timefreeze(get_turf(user))
+	..()
+
+/obj/item/slimecross/burning/cerulean
+	colour = "cerulean"
+
+/obj/item/slimecross/burning/cerulean/do_effect(mob/user)
+	user.visible_message("<span class='notice'>[src] produces a potion!</span>")
+	new /obj/item/slimepotion/extract_cloner(get_turf(user))
+	..()
+
+/obj/item/slimecross/burning/pyrite
+	colour = "pyrite"
+
+/obj/item/slimecross/burning/pyrite/do_effect(mob/user)
+	user.visible_message("<span class='danger'>[src] releases a colorful wave of energy, which shatters the lights!</span>")
+	var/area/A = get_area(user.loc)
+	for(var/obj/machinery/light/L in A) //Shamelessly copied from the APC effect.
+		L.on = TRUE
+		L.break_light_tube()
+		L.on = FALSE
+		stoplag()
+	..()
+
+/obj/item/slimecross/burning/red
+	colour = "red"
+
+/obj/item/slimecross/burning/red/do_effect(mob/user)
+	user.visible_message("<span class='danger'>[src] pulses a hazy red aura for a moment, which wraps around [user]!</span>")
+	for(var/mob/living/simple_animal/slime/S in view(7, get_turf(user)))
+		if(user in S.Friends)
+			var/friendliness = S.Friends[user]
+			S.Friends = list()
+			S.Friends[user] = friendliness
+		else
+			S.Friends = list()
+		S.rabid = 1
+		S.visible_message("<span class='danger'>The [S] is driven into a dangerous frenzy!</span>")
+	..()
+
+/obj/item/slimecross/burning/green
+	colour = "green"
+
+/obj/item/slimecross/burning/green/do_effect(mob/user)
+	var/which_hand = "l_hand"
+	if(!(user.active_hand_index % 2))
+		which_hand = "r_hand"
+	var/mob/living/L = user
+	if(!istype(user))
+		return
+	var/obj/item/held = L.get_active_held_item() //This should be itself, but just in case...
+	L.dropItemToGround(held)
+	var/obj/item/nullrod/armblade/slime/blade = new(user)
```

Comment:
Should probably be changeling one subtype, nullrod has some implications that don't make much sense here.